libvast/CMakeLists.txt

```diff
@@ -244,7 +244,7 @@ if (VAST_USE_OPENSSL)
   target_link_libraries(libvast PUBLIC caf::openssl)
 endif ()
 
-target_link_libraries(libvast PRIVATE flatbuffers::flatbuffers)
+target_link_libraries(libvast PUBLIC flatbuffers::flatbuffers)
```

Comment:
Is there an unavoidable reason that flatbuffers leak into the interface here?

Reply:
Yeah, zeek-to-vast doesn't build without it. I'd use the INTERFACE target, but that's broken unless we bump the CMake version (a lot).